elm/spec/dependabot/elm/file_updater_spec.rb

```diff
@@ -69,58 +62,6 @@
   describe "#updated_dependency_files" do
     subject(:updated_files) { updater.updated_dependency_files }
 
-    it "doesn't store the files permanently" do
-      expect { updated_files }.to_not(change { Dir.entries(tmp_path) })
-    end
-
-    it "returns DependencyFile objects" do
-      updated_files.each { |f| expect(f).to be_a(Dependabot::DependencyFile) }
-    end
-
-    it { expect { updated_files }.to_not output.to_stdout }
-    its(:length) { is_expected.to eq(1) }
```

Comment:
Should we bring these back? Look like they are just testing the shape of `updated_dependency_files`

Reply:
They weren't testing the code that we've been running for most users in production. Moving tests over proved to be fairly painful after having done a few for the `UpdateChecker`, I mostly considered it as "also wasn't tested before" and was unsure it was worth spending more time on this given the low usage.

Reply:
Aye, yeah this was pretty easy to patch up, good call-out 👍